tests/units/test_catalog.py

```diff
@@ -284,16 +284,37 @@ def test__tests_setter_fail(self, catalog_data: dict[str, Any]) -> None:
             catalog.tests = catalog_data["tests"]
         assert catalog_data["error"] in str(exec_info)
 
-    def test_get_tests_by_tags(self) -> None:
-        """Test AntaCatalog.get_tests_by_tags()."""
+    def test_build_indexes_all(self) -> None:
```

Comment:
there should be new tests for the runner

Reply:
Added in 88a2f83a6d6b93d91f0c8ca3bc5a40a36df5eb68